### ISSUE TITLE
BUG - 2 - Android - Accessibility: Screen reader skipping over certain content within What's New notification box

### ISSUE THREAD

**Issue by SKMOstudio** [department-of-veterans-affairs/va-mobile-app#12119]
## What happened?

When a screen reader user attempts to navigate through the "What's New" notification box, not all of the elements within the box are easily focusable. Based on the formatting of the content, the screen reader is skipping over certain content entirely.

## Specs:

- Device: Samsung Galaxy S24
- OS: Android
- User Account: DEMO
- Accessibility State: Screen reader ON, default settings.

## Steps to Reproduce

Setup: Ensure the **remoteConfigRefreshTest** is ON and the What's New message has not been dismissed.
1. Log into the app as the DEMO user. Ensure the screen reader is ON.
2. Focus on the What's New notification banner and expand it.
3. While focused on the What's New notification, attempt to swipe left/right to navigate through the content within it.

## Acceptance Criteria

When a screen reader user focuses on the What's New notification box, they should be able to navigate easily through the header, description, and other body content displayed within it.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High **Low** 
- Frequency: **High** Low 

- 3 - Low
- 2 - **High**
- 1 - Critical

## Screen shot(s) and additional information

https://github.com/user-attachments/assets/437825f7-d31b-49d2-861d-7a787be3d900

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by matthew-df** [comment]
@dumathane I took a look at this and found a minimal change to the component library that resolves this issue. Wrapping the description and children values in an accessible prevents the screen reader from highlighting the entire box because it now has correct points to go to. This is a much less invasive change than my previous approach of disabling accessibility for all other wrapper views.
```
  {description ? (
    <View accessible={true} >  // New wrapper View
      <Text
        variant="body"
        size="lg"
        bottomSpacing="none"
        a11yLabel={descriptionA11yLabel || description}>
        {description}
      </Text>
    </View>
  ) : null}
  {description && children ? <Spacer size="lg" /> : null}
  <View accessible={true}>  // New wrapper View
    {children}
  </View>
```
